docs/general/events.md

```diff
@@ -43,18 +43,10 @@ that identify the class of Events but not the instance of the Event.
 <!-- semconv event -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
-| `event.domain` | string | The domain identifies the business context for the events. [1] | `browser` | Required |
-| `event.name` | string | The name identifies the event. | `click`; `exception` | Required |
+| [`event.domain`](../attributes-registry/events.md) | string | The domain identifies the business context for the events. [1] | `browser` | Required |
+| [`event.name`](../attributes-registry/events.md) | string | The name identifies the event. | `click`; `exception` | Required |
 
 **[1]:** Events across different domains may have same `event.name`, yet be unrelated events.
-
```

Comment:
Please render the table with `full` (check other examples) so we don't loose the enum table values

Reply:
as `event.domain` has been removed it looks like we don't need the `full` render anymore